EG23.tex

```diff
@@ -180,30 +195,35 @@ \subsection{Reactors}
 %% Apropos my comment above - this is a great example of showing others how to
 %% translate specifications into Cyclus inputs.
```

Comment:
I am not sure to understand what you mean there.

(The breeding ratio were only specify in the recipe which are in table 2.)